AUTHORS.md

```diff
@@ -2,12 +2,12 @@
 
 We list here the main contributors to this specification:
 
-* Stefano Pullini <stefano.pullini@smc.it>
 * Giuseppe De Marco <giuseppe.demarco@teamdigitale.governo.it>
-* Michele D'Amico <>
-* Francesco Antonio Marino <>
-* Antonio Colella <>
-* Nunzio Napolitano <>
-* Antonio Florio <>
+* Michele D'Amico <michele.damico@agid.gov.it>
+* Francesco Antonio Marino <fa.marino@ipzs.it>
+* Antonio Colella <colella@agid.gov.it>
+* Nunzio Napolitano <nunzio.napolitano@agid.gov.it>
+* Antonio Florio <florio@agid.gov.it>
+* Stefano Pullini <stefano.pullini@smc.it>
 
```

Comment:
Vi chiederei di aggiungere come contributors anche 
- Giada Sciarretta <giada.sciarretta@fbk.eu>
- Amir Sharif <asharif@fbk.eu>

Hanno contribuito in modo importante alla stesura delle specifiche CIE nell'ambito della collaborazione con Futuro&Conoscenza (controllata IPZS)

Reply:
Done 6d58593cd38dec4fa606742e06f8a98ddf11635f